CoreFoundation/Base.subproj/CFRuntime.c

```diff
@@ -1351,6 +1387,8 @@ int DllMain( HINSTANCE hInstance, DWORD dwReason, LPVOID pReserved ) {
 	// do these last
 	if (cfBundle) CFRelease(cfBundle);
         __CFStringCleanup();
+    } else if (dwReason == DLL_THREAD_DETACH) {
+        __CFFinalizeWindowsThreadData();
```

Comment:
This code was removed in #2902, as we are using modern thread local storage mechanics in Windows, which doesn't need explicit finalization.

Reply:
Yup. The Windows build isn't erroring out at the undefined function; will remove.